src/flask-api/net_predictor/WR_logistic_regression.py

```diff
@@ -0,0 +1,119 @@
+import pandas as pd
+from sklearn.linear_model import LogisticRegression
+from sklearn import metrics 
+import kenpom_creds as cred # A file (named "kenpom_creds.py") with proper credentials must be made in the "net-predictor" folder 
+from kenpompy.utils import login
+import kenpompy.summary as kpsum
+import kenpompy.misc as kpmisc
+import kenpompy.team as kpteam
+
+browser = login(cred.login_email, cred.login_password)
+training_season = 2020  # Choose which year's data the model is trained on
+testing_season = 2021   # Choose which year's data the model is tested on
+# team_name = 'Loyola Marymount'
+
+valid_teams_train = kpteam.get_valid_teams(browser, season=training_season)
+valid_teams_test = kpteam.get_valid_teams(browser, season=testing_season)
+
+stats_train = kpsum.get_efficiency(browser, season=training_season)
+stats_test = kpsum.get_efficiency(browser, season=testing_season)
+columns_to_drop = ['Result', 'Opponent Name'] ### TODO: figure out what Location_semi-away and home is ###
+label_name = 'Result'
+
+complete_data_train = pd.DataFrame()
+complete_data_test = pd.DataFrame()
+
+valid_teams_train = [team for team in valid_teams_train if team[-1] != '*' and '&' not in team]
+valid_teams_test = [team for team in valid_teams_test if team[-1] != '*' and '&' not in team]
+
+for team_name in valid_teams_train:
+  team_win_loss_train = kpteam.get_schedule(browser, team=team_name, season=training_season)
+  # team_win_loss_test = kpteam.get_schedule(browser, team=team_name, season=testing_season)
+
+  # Get a teams schedule by getting a dataframe with whether or not they won or lost, and the name of their opponent
+  team_win_loss_train = team_win_loss_train[team_win_loss_train['Opponent Name'].isin(valid_teams_train)][['Result', 'Opponent Name', 'Location']]
+  team_win_loss_train['Result'] = team_win_loss_train['Result'].apply(lambda x: x[:1])
+  team_win_loss_train['Result'] = team_win_loss_train['Result'].replace({'W': 1, 'L': 0})
+  team_win_loss_train = pd.get_dummies(team_win_loss_train, columns=['Location']) # One-Hot Encoding
+
+  # team_win_loss_test = team_win_loss_test[team_win_loss_test['Opponent Name'].isin(valid_teams)][['Result', 'Opponent Name', 'Location']]
```

Comment:
Remove comments before committing to reduce clutter and make future use development easier.
https://kentcdodds.com/blog/please-dont-commit-commented-out-code